src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el.length > 0)
+    .map((el) => el.split(':'))
+    .reduce((acc, curr) => {
+      acc[curr[0].trim()] = curr[1].trim();
```

Comment:
```suggestion
    .reduce((acc, curr) => {
      const [key, value] = curr;
      acc[key.trim()] = value.trim();
```
You can use destructing like this.
Also, a variable name `el` isn't so good, it could be `property`